packages/create/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "create-keystone-app",
+  "version": "9.0.1",
+  "license": "MIT",
+  "type": "module",
+  "main": "dist/create-keystone-app.cjs.js",
+  "module": "dist/create-keystone-app.esm.js",
+  "repository": "https://github.com/keystonejs/keystone/tree/main/packages/create",
+  "bin": "./cli.js",
+  "exports": {
+    ".": {
+      "module": "./dist/create-keystone-app.esm.js",
+      "default": "./dist/create-keystone-app.cjs.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "preconstruct": {
+    "entrypoints": [
+      "index.ts"
+    ]
+  },
```

Comment:
Copied the `package.json` from the `auth` module and adjusted it to work with this one.
Also marked as `"type": "module"`